client/src/GameInformation/GameInformation.css

```diff
@@ -4,20 +4,11 @@
 	font-size: large;
 }
 
-.gameId {
-	border: 10px solid white;
-}
-
-.shareText {
-	border: 10px solid white;
-}
-
-.playerColour {
-	border: 10px solid white;
-}
-
+.gameId,
+.shareText,
+.playerColour,
 .whoIsNext {
-	border: 10px solid white;
+	margin: 10px;
```

Comment:
You don't seem to be using the borders for any reason (white on white) so you must be using them for positioning/spacing and that's what margin is intended for.

I suppose could have changed `white` to `transparent` but a transparent border makes no sense.

I does change the layout marginally because the `box-sizing` is set to `border-box` which means that any width/height given have to include border and padding, but not margin.  The alternative and default is `content-box` where the width/height is just the content and padding/margin are added on but that's a lot harder to work with.